enterprise_access/apps/api/serializers/content_assignments/assignment.py

```diff
@@ -51,6 +51,16 @@ class LearnerContentAssignmentRecentActionSerializer(serializers.Serializer):
     )
 
 
+class ContentMetadataSerializer(serializers.Serializer):
+    """
+    Serializer to help return additional content metadata for assignments.  These fields should
+    map more or less 1-1 to the fields in content metadata dicts returned from the
+    enterprise-catalog `get_content_metadata` response payload.
+    """
```

Comment:
This gives us a common place to hook into any additional data we want from the enterprise-catalog `get_content_metadata` endpoint.

Reply:
I like this pattern of having an isolated place to extract specific fields from `get_content_metadata` endpoint as needed.